docs/community/governance.md

```diff
@@ -0,0 +1,180 @@
+
+Team members may retire at any time by emailing [observatorium-team@googlegroups.com][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [observatorium-team@googlegroups.com][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+### Technical decisions
+
+Smaller technical decisions are made informally and [lazy consensus](#consensus) is assumed. Technical decisions that span multiple parts of the Observatorium project
+should be discussed and made on the [GitHub issues][issues] and in most cases followed by proposal as described [here](/CONTRIBUTING.md#adding-new-features--components).
+
+Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Governance changes
+
+Material changes to this document are discussed publicly on the [Observatorium GitHub](http://github.com/observatorium/observatorium).
+Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
+
+### Other matters
+
+Any matter that needs a decision, including but not limited to financial matters, may be called to a vote by any Maintainer if they deem it necessary.
+For financial, private, or personnel matters, discussion and voting takes place on the [observatorium-team@googlegroups.com][team]; Otherwise discussion and votes are held in public on the GitHub issues or #observatorium-dev CNCF slack channel.
+
+## Voting
+
+The Observatorium project usually runs by informal consensus, however sometimes a formal decision must be made.
+
+Depending on the subject matter, as laid out [above](#decision-making), different methods of voting are used.
+
+For all votes, voting must be open for at least one week. The end date should be clearly stated in the call to vote.
+A vote may be called and closed early if enough votes have come in one way so that further votes cannot change the final decision.
+
+In all cases, all and only [Maintainers](#maintainers-team) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personnel matters (including but not limited to team membership and maintainership) are held in private on the [observatorium-team@googlegroups.com][team]. All other discussion and votes are held in public on the GitHub issues or #observatorium-dev CNCF slack channel.
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [Maintainers Team](#maintainers-team).
+
+### Governance
+
+It's important for the project to stay independent and focused on shared interest instead of single use case of one company or organization.
```

Comment:
```suggestion
It's important for the project to stay independent and focused on shared interests instead of on the single use case of one company or organization.
```